common/src/main/java/org/conscrypt/HpkeContextRecipient.java

```diff
@@ -76,25 +77,25 @@ public static HpkeContextRecipient setupBase(
      *                               most likely if the keys configured are not valid)
      */
     public byte[] open(byte[] ciphertext, byte[] aad) {
```

Comment:
The javadoc might need to be updated, mentioning the `init` method needs to be called first.